web-extension/_locales/en/messages.json

```diff
@@ -88,6 +88,10 @@
     "message": "To copy the secret to clipboard, hold shift key while clicking on the secret.",
     "description": "Clipboard hint text"
   },
+  "optionsUsageHintsDetailsText": {
+    "message": "To open the metadate for an entry, hold alt key while clicking on the secret.",
```

Comment:
Typo "metadate". Also I don't think "metadata" is a common enough word to use here.
Maybe just "To show details for an entry [...]".